transbank/webpay/webpayplus/transaction.py

```diff
@@ -0,0 +1,161 @@
+import json
+
+from transbank.webpay.webpayplus.transaction_create_response import TransactionCreateResponse
+from transbank.webpay.webpayplus.transaction_commit_response import TransactionCommitResponse
+from transbank.webpay.webpayplus.transaction_refund_response import TransactionRefundResponse
+from transbank.webpay.webpayplus.transaction_status_response import TransactionStatusResponse
+from transbank.webpay.webpayplus.webpayplus import *
+
+
+class Transaction:
+    CREATE_TRANSACTION_ENDPOINT = 'rswebpaytransaction/api/webpay/v1.0/transactions'
+    COMMIT_TRANSACTION_ENDPOINT = 'rswebpaytransaction/api/webpay/v1.0/transactions'
+    REFUND_TRANSACTION_ENDPOINT = 'rswebpaytransaction/api/webpay/v1.0/transactions/{0}/refunds'
+    GET_TRANSACTION_STATUS_ENDPOINT = 'rswebpaytransaction/api/webpay/v1.0/transactions/{0}'
+
+    def __init__(self):
+        pass
+
+    @classmethod
+    def create(cls, buy_order, session_id, amount,
+               return_url, options=None):
+        commerce_code = WebpayPlus.commerce_code()
+        api_key = WebpayPlus.api_key()
+        base_url = WebpayPlus.integration_type_url()
+
+        if options is not None:
+            commerce_code = options.commerce_code
+            api_key = options.api_key
+            base_url = WebpayPlus.integration_type_url(options.integration_type)
+
+        headers = dict({
```

Comment:
You don't need to use `dict` when using the literal dict notation. `dict({'foo': 'bar'))` is a longer/slower version of `{'foo': 'bar'}`

Reply:
So, let me do the change! Thanks for your comments

Reply:
Let me change it